src/components/YextEntityFieldSelector.tsx

```diff
@@ -8,7 +8,7 @@ import {
 } from "../utils/getFilteredEntityFields.ts";
 
 export type EntityFieldType = {
-  fieldName: string;
+  fieldId: string;
```

Comment:
This will change again per the updated [spec](https://docs.google.com/document/d/1qB8SRHidaEnAqJWiwIqptju3ZVTGkl3fl5cK50-cNJE/edit#heading=h.cr650zdk0xpg) so maybe we hold off until that.